app/src/main/java/com/jbnm/homehero/ui/goal/GoalActivity.java

```diff
@@ -1,30 +1,53 @@
 package com.jbnm.homehero.ui.goal;
 
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.app.DialogFragment;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.content.Intent;
 import android.graphics.Color;
 import android.os.Bundle;
 import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.animation.Animation;
+import android.view.animation.AnimationUtils;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
 import android.widget.ImageView;
+import android.widget.ListAdapter;
+import android.widget.TextView;
 
 import com.hookedonplay.decoviewlib.DecoView;
 import com.hookedonplay.decoviewlib.charts.EdgeDetail;
 import com.hookedonplay.decoviewlib.charts.SeriesItem;
 import com.hookedonplay.decoviewlib.charts.SeriesLabel;
 import com.hookedonplay.decoviewlib.events.DecoEvent;
 import com.jbnm.homehero.R;
-import com.jbnm.homehero.data.model.Child;
+import com.jbnm.homehero.data.model.Reward;
+import com.jbnm.homehero.ui.taskpicker.TaskPickerActivity;
+import com.jbnm.homehero.ui.taskprogress.TaskProgressActivity;
+
+import java.util.List;
 
 import butterknife.BindView;
 import butterknife.ButterKnife;
 
+import static com.jbnm.homehero.R.array.replace_me;
+
 /**
  * Created by nathanielmeyer on 7/18/17.
  */
 
 public class GoalActivity extends AppCompatActivity implements GoalContract.MvpView {
```

Comment:
If GoalActivity extends BaseActivity you can get rid of showLoading, hideLoading, showError and hideError since they are already implemented in BaseActivity